.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,50 @@
+name: Lint
```

Comment:
`ci` is probably a better name for this file than `golangci-lint`. Reason is now you can ONLY have linting in this file. Which means you’d have to set up entirely different files for testing, PR checks, and any other CI workflow actions. All while copying + pasting more and more config that may diverge between them all.

I’d probably keep the name `ci.yaml` as before—then you can add additional jobs that all share setup.

Don’t create more work for yourself until you absolutely need to!

Reply:
Also this is unnecessary but I’m just marking convos as “resolved” when they are done. Totally skippable, but when PRs get really noisy it helps!